akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala

```diff
@@ -251,15 +255,48 @@ object Polls {
     }
   }
 
-  private def pollResultToWhiteboardShape(result: SimplePollResultOutVO): scala.collection.immutable.Map[String, Object] = {
+  private def pollResultToWhiteboardShape(result: SimplePollResultOutVO, page: PresentationPage): scala.collection.immutable.Map[String, Object] = {
+    val maxImageWidth = 1440
```

Comment:
@KDSBrowne do we need to add a check here if we're dealing with infinite wb? Or the poll always publishes over the initial area regardless of how much you've expanded?